src/workerd/api/node/tests/http-client-nodejs-server.js

```diff
@@ -92,6 +92,17 @@ const helloWorldServer = http.createServer((req, res) => {
       ]);
       res.end();
       break;
+    case 'echo':
+      // Echo the request body back as the response
```

Comment:
fwiw, I believe you should be able to just do...

```
import { pipeline } from 'node:streams';

// ...

await pipeline(req, res);
```

Reply:
TIL